cmd/repl/prompt.go

```diff
@@ -287,12 +312,16 @@ var consoleWriter = prompt.NewStdoutWriter()
 
 func runPrompt(cb func(string)) {
 	state := newPromptState()
+	history := []string{}
+	for _, h := range state.history {
+		history = append([]string{h.Text}, history...)
+	}
```

Comment:
Revert `state.history`?

Reply:
Yes，because `state.history` is loaded reversely to make the history completion show recent commands first.